docs/conf.py

```diff
@@ -170,13 +167,25 @@ def generate_parameter_table(input_csv_file, output_csv_file):
     )
 
 
-generate_parameter_description("MVS_parameters_list.csv", "MVS_parameters_list.inc")
-generate_parameter_table("MVS_parameters_list.csv", "MVS_parameters_list.tbl")
+def copy_readme():
+    with open("../README.rst", "r") as fp:
```

Comment:
```suggestion
    with open("../README.rst", "r", encoding="utf8") as fp:
```

@Bachibouzouk this has now worked for windows!